go/vt/topo/server.go

```diff
@@ -181,6 +186,9 @@ var (
 
 	FlagBinaries = []string{"vttablet", "vtctl", "vtctld", "vtcombo", "vtgate",
 		"vtorc", "vtbackup"}
+
+	// Default read concurrency to use in order to avoid overhwelming the topo server.
+	DefaultReadConcurrency int64 = 32
```

Comment:
Moved + renamed from `DefaultConcurrency` from `keyspace.go`

`DefaultReadConcurrency` seems more accurate as this only applies to reads